README.md

```diff
@@ -12,6 +12,10 @@ Read more about the [IAMC data format](https://docs.ece.iiasa.ac.at/iamc.html).
 This project uses the Python package [nomenclature-iamc](https://nomenclature-iamc.readthedocs.io)
 for management of codelists and validation of scenario data in the IAMC data format.
 
+For *users not comfortable working with GitHub repositories and yaml files*,
```

Comment:
You could highlight this using GitHub markdown alerts https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts using _tip_ or _important_ for example

Reply:
Beautiful!